ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -105,6 +105,12 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 							Optional:    true,
 							Description: "Account ID of KMS instance holder - if not provided, defaults to the account in use",
 						},
+						"wait_for_apply": {
```

Comment:
We have some kind of similar parameter "wait_till" which monitor to wait for certain status will n't any of those status make sure we should wait till KMS is applied to master?

Reply:
Having multiple wait status parameter may be kind of confusing parameters to end user's

wait_for_apply is a kind of bool parameter we are waiting for kms tomorrow if we need to wait for any other specific service ?

Reply:
I was thinking about it, but having different waits at different locations, i think would just mean that all applies. They do. And this way is it can be a combination of waits, while in the singe wait_till we would have to add some extra logic, or a whole lot of options to cover.